operations/multi-tenancy/ExampleWorkflow.md

```diff
@@ -0,0 +1,755 @@
+    ```
+
+* (`ncn-mw#`) Extract the slurm.conf:
+
+    ```bash
+    yq r nocls01a-slurm-conf.yaml 'data."slurm.conf"' > slurm.conf
+    ```
+
+* (`ncn-mw#`) Edit the slurm.conf:
+    * See Appendix C
+
+    ```bash
+    vi slurm.conf
+    ```
+
+* (`ncn-mw#`) Update the configuration:
+
+    ```bash
+    yq w -i nocls01a-slurm-conf.yaml 'data."slurm.conf"' "$(cat slurm.conf)"
+    ```
+
+* (`ncn-mw#`) Apply the configuration:
+
+    ```bash
+    kubectl apply -f nocls01a-slurm-conf.yaml
+    ```
+
+* (`ncn-mw#`) Look up the pod for the tenant `slurmcluster`:
+
+    ```bash
+    SLURMCTLD_POD=$(kubectl get pod -n vcluster-noten01a-slurm -lapp=slurmctld -o name)
+    ```
+
+* (`ncn-mw#`) Reconfigure:
+
+    ```bash
+    kubectl exec -n user ${SLURMCTLD_POD} -c slurmctld -- scontrol reconfigure
+    ```
+
+Repeat this step as needed for additional `Slurmclusters`.
+
+### Configure USS group variables
+
+Filename:  `group_vars/noten01a/slurm.yml`
+
+* (`ncn-mw#`) Clone the USS repository:
+
+    ```bash
+    git clone https://api-gw-service-nmn.local/vcs/cray/uss-config-management.git
+    ```
+
+* (`ncn-mw#`) Go to repo:
+
+    ```bash
+    cd uss-config-management
+    ```
+
+* (`ncn-mw#`) Check out integration branch (1.1.0 shown here):
+
+    ```bash
+    git checkout integration-1.1.0
+    ```
+
+* (`ncn-mw#`) Create subdirectory for tenant:
+
+    ```bash
+    mkdir group_vars/noten01a
+    ```
+
+* (`ncn-mw#`) Edit the file `group_vars/noten01a/slurm.yml`
+    * See an appropriate example in Appendix D
+
+* (`ncn-mw#`) Add the new file:
+
+    ```bash
+    git add group_vars/noten01a/slurm.yml
+    ```
+
+* (`ncn-mw#`) Commit the new file:
+
+    ```bash
+    git commit -am "descriptive comment"
+    ```
+
+* (`ncn-mw#`) Push to integration branch (1.1.0 shown here):
+
+    ```bash
+    git push origin integration-1.1.0
+    ```
+
+* (`ncn-mw#`) Remember the first commit ID in the output:
+
+    ```bash
+    git log -a |cat
+    ```
+
+    Repeat this step as needed for additional tenants.
+
+### Create BOS session template
+
+Note that you will need one template for each node type (UAN,Compute) and architecture (X86,ARM) in the tenants.
+You can use a single BOS session template for many tenants of the same node type and architecture.
+
+* (`ncn-mw#`) Look up the name of the default template(s) for tenants (e.g. X86 Compute) and save as JSON file(s):
+
+    ```bash
+    cray bos sessiontemplates describe --format json ssi-compute-cr_2024.x86_64-cr_2024_1 > ssi-compute-cr_2024.x86_64-cr_2024_1.json
+    ```
+
+* (`ncn-mw#`) Make a copy of the default template:
+
+    ```bash
+    cp ssi-compute-cr_2024.x86_64-cr_2024_1.json ssi-compute-cr_2024.x86_64-cr_2024_1-tenants.json
+    ```
+
+* Edit the new copy:
+    * Delete block of lines starting with `enable_cfs:`, `name:`, and `tenant:` and remove the comma from the preceding line
+    * Change the name of the CFS configuration to include the tenant name
```

Comment:
Change the name of the CFS configuration in the BOS session template to add a '-tenants' suffix, as seen in the next section

Reply:
fixed